examples/binaries/calc-hash/aggregator/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "demo-calc-hash-aggregator"
+version = "0.1.0"
+authors = ["Gear Technologies"]
+edition = "2018"
+license = "GPL-3.0"
+workspace = "../../../../"
+
+[dependencies]
+gstd = { path = "../../../../gstd", features = ["debug"] }
+
+# encoding
+codec = { package = "parity-scale-codec", version = "3.1.2", default-features = false, features = ["derive"]}
+shared = { path = "../", package = "demo-calc-hash", default-features = false }
+
+[build-dependencies]
+gear-wasm-builder = { path = "../../../../utils/wasm-builder" }
+
+[features]
+std = [
+  "shared/std",
+  "codec/std",
+]
+default = [ "std" ]
```

Comment:
why do we need this by default?

Reply:
can not build WASM_BINARY without this, each of examples/binaries/* have this